src/languages/python.js

```diff
@@ -392,14 +392,14 @@ export default function(hljs) {
           {
             match: [
               /class/, /\s+/,
-              IDENT_RE, /\s*/,
-              /\(\s*/, IDENT_RE,/\s*\)/
+              UNDERSCORE_IDENT_RE, /\s*/,
+              /\(\s*/, UNDERSCORE_IDENT_RE,/\s*\)/
             ],
           },
           {
             match: [
               /class/, /\s+/,
-              IDENT_RE
+              UNDERSCORE_IDENT_RE
```

Comment:
Is this just for completeness?  Are classes often named in such a fashion?  I thought convention was `CamelCase`?

Reply:
Class names in Python follow the global identifiers specification, so yes class names starting with an underscore are totally allowed by the language, see example [here](https://forge.softwareheritage.org/source/swh-web/browse/master/swh/web/tests/conftest.py$170).